packages/core/src/create/release.ts

```
@@ -104,6 +93,7 @@ export const createReleaseJson = async (
     },
     extensions: {
       solana_dapp_store: {
+        // @ts-expect-error
```

can you comment what error is being suppressed here and why?